### PR TITLE
Dev entityrdfizer

### PR DESCRIPTION
**Pull request by paul-cares**
Improvement of the abox-writer.
Key features:
- user warnings for invalid data in the input csv file
- comparison of the abox entities to the tbox specification (optional)
- extended user instructions (the Readme file)

### CODE REVIEW

File: EntityRDFizer/README.md
Comment (msff2) [inline EntityRDFizer/README.md:1]: While running 'pytest', I experienced the following errors. Please fix the issue. You may need to replace the old OWL files with new ones.

============================================================================== short test summary info ===============================================================================
FAILED tests/test_entrdifizer.py::test_csv2abox[ontospecies-ontospecies_abox.csv-False] - UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb1 in position 2019: invalid start byte
FAILED tests/test_entrdifizer.py::test_csv2abox[ontocropenergy-ontocropenergy_abox.csv-False] - assert '<rdf:type rd...owl/Barley"/>' == '<rdf:type rd...owl#Barley"/>'
FAILED tests/test_entrdifizer.py::test_csv2abox[ontokgrouter-ontokgrouter_abox.csv-False] - assert '<rdf:type rd...etResource"/>' == '<rdf:type rd...etResource"/>'
FAILED tests/test_entrdifizer.py::test_csv2abox[ontolanduse-ontolanduse_abox.csv-False] - UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa0 in position 1567: invalid start byte
FAILED tests/test_entrdifizer.py::test_csv2abox[ontocompchem-ontocompchem_abox.csv-False] - UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa0 in position 1342: invalid start byte
FAILED tests/test_entrdifizer.py::test_csv2tbox[ontospecies-ontospecies_tbox.csv-False] - TypeError: run_tbox_generator() takes 2 positional arguments but 3 were given
FAILED tests/test_entrdifizer.py::test_csv2tbox[ontocropenergy-ontocropenergy_tbox.csv-False] - TypeError: run_tbox_generator() takes 2 positional arguments but 3 were given
FAILED tests/test_entrdifizer.py::test_csv2tbox[ontokgrouter-ontokgrouter_tbox.csv-False] - TypeError: run_tbox_generator() takes 2 positional arguments but 3 were given
FAILED tests/test_entrdifizer.py::test_csv2tbox[ontolanduse-ontolanduse_tbox.csv-False] - TypeError: run_tbox_generator() takes 2 positional arguments but 3 were given
FAILED tests/test_entrdifizer.py::test_csv2tbox[ontocompchem-ontocompchem_tbox.csv-False] - TypeError: run_tbox_generator() takes 2 positional arguments but 3 were given
================================================================================= 10 failed in 0.69s =================================================================================

**Review by msff2 — COMMENTED** [review]
Please address my comments on 'pytest' and some syntactical modifications in the README.md file.

File: EntityRDFizer/README.md
Comment (paul-cares) [reply EntityRDFizer/README.md:1]: Fixed the first type of error (encoding). Now file can be in either utf-8 or default OS encoding.

Comment (paul-cares) [reply EntityRDFizer/README.md:1]: Fixed the second type of error (extra argument in tbox).

Comment (msff2) [reply EntityRDFizer/README.md:1]: Alex updated the input CSV files to reduce the size of each file to 5 KB, generated the reference OWL files, and pushed the changes. After his changes, all the tests passed.

**Pull request merged**